### DISCUSSION TITLE
How to get some sample data in the local database

### DISCUSSION THREAD

**Discussion by joe-shajan** [EddieHubCommunity/LinkFree#4766] [Q&A] [answered]
So right now I'm working search. So in order to test and debug there are no users or any other data in the database when we install the project. So I just want to know what others do in this scenario or is there any way to get the data from the production database?

**Comment by Vijaykv5** [comment]
Suppose  you don't have any data in your local development database, you can use these  methods to get some sample data into it:

1. **Create dummy data** : You can manually create some sample data in your local development database to use for testing and debugging. This can be a good option if you only need a small amount of data or if you need very specific data for your testing.

2. **Use seed data**: Many frameworks and libraries have the ability to seed a database with pre-defined data. This can be a good option if you need a larger amount of data or if you want to have more realistic data to work with.

3. **Import data from a production database**: If you have access to a production database, you can export the data and import it into your local development database. If you need lot of data , then it is a good approach .

Use publicly available datasets: For example, you can use the Stack Overflow public data  to create a database of questions, answers, and users to use for testing.

Hope it would be helpful :)

**Comment by eddiejaoude** [comment, marked as answer]
You can visit the loading data api `/api/system/reload?secret=development` more info can be found in our docs https://www.biodrop.io/docs/environments/local-development
